src/components/pacbio/PacbioPoolTubeItem.vue

```diff
@@ -8,35 +8,51 @@
   >
     <b-row>
       <b-col cols="3">
-        <b-img src="/tube.png" :draggable="!!valid" />
+        <b-img src="/tube.png" />
+        <b-button
+          :id="`editPool-${id}`"
+          size="sm"
+          variant="outline-primary"
+          class="btn-block"
+          :to="{ name: 'PacbioPoolCreate', params: { id: id } }"
+          >Edit</b-button
+        >
       </b-col>
       <b-col cols="9">
         <dl class="row">
           <dt>Barcode</dt>
           <dd data-attribute="barcode">{{ barcode }}</dd>
         </dl>
-        <dl v-if="showInfo" class="row">
-          <dt>Source</dt>
-          <dd data-attribute="source-identifier">{{ source_identifier }}</dd>
-          <dt>Volume</dt>
-          <dd data-attribute="volume">{{ volume || 'Unknown' }}</dd>
-          <dt>Concentration</dt>
-          <dd data-attribute="concentration">{{ concentration || 'Unknown' }}</dd>
-          <dt>Template prep kit box barcode</dt>
-          <dd data-attribute="template-prep-kit-box-barcode">
-            {{ template_prep_kit_box_barcode || 'Unknown' }}
-          </dd>
-          <dt>Insert size</dt>
-          <dd data-attribute="insert-size">{{ insert_size || 'Unknown' }}</dd>
-          <dt>Libraries</dt>
-          <dd>
-            <ul>
-              <li v-for="library in libraries" :key="library.id">
-                {{ library.sample_name }} : {{ library.group_id }}
-              </li>
-            </ul>
-          </dd>
-        </dl>
+        <div v-if="showInfo">
+          <dl class="row">
+            <dt>Source</dt>
+            <dd data-attribute="source-identifier">{{ source_identifier }}</dd>
+            <dt>Volume</dt>
+            <dd data-attribute="volume">{{ volume || 'Unknown' }}</dd>
+            <dt>Concentration</dt>
+            <dd data-attribute="concentration">{{ concentration || 'Unknown' }}</dd>
+            <dt>Template prep kit box barcode</dt>
+            <dd data-attribute="template-prep-kit-box-barcode">
+              {{ template_prep_kit_box_barcode || 'Unknown' }}
+            </dd>
+            <dt>Insert size</dt>
+            <dd data-attribute="insert-size">{{ insert_size || 'Unknown' }}</dd>
+            <dt>Libraries</dt>
+            <dd>
+              <ul>
+                <li v-for="library in libraries" :key="library.id">
+                  {{ library.sample_name }} : {{ library.group_id }}
+                </li>
+              </ul>
+            </dd>
+            <dt v-if="!valid">Errors</dt>
+            <dd v-if="!valid">
+              <ul>
+                <li v-for="(error, index) in errors" :key="index">{{ error }}</li>
+              </ul>
+            </dd>
+          </dl>
```

Comment:
We include any errors on the pool. This is especially useful if it is one of the libraries which is incomplete, as that information is not immediately visible.